### PR TITLE
chore: update side-nav scroll styles and remove scroll indicator click behavior

### PR DESCRIPTION
**Pull request by SamanthaAO**
# Description

- update scroll indicator and icon styles (side-nav) to more closely align with figma
- remove click smooth scroll from scroll indicators
- add changes to improve the side navs alignments with each other compared to the original version of this PR which was intended for feature/fast-frame

scroll-indicator

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts [hunk 1]
```diff
@@ -2,56 +2,54 @@ import { css } from "@microsoft/fast-element";
...
+    accentForegroundCutRestBehavior,
     accentForegroundHoverBehavior,
-    neutralForegroundRestBehavior,
 } from "@microsoft/fast-components";
 
 export const SideNavigationStyles = css`
-    ${display("flex")}:host {
-        --engaged-height: calc(var(--design-unit) * 9px);
+    ${display("flex")} :host {
+        --active-height: calc(var(--design-unit) * 13px);
```
Comment (chrisdholt) [inline sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts:11]: The references to --design-unit here seem somewhat arbitrary in terms of what we are multiplying by. Do we want this to change size in coordination with the design unit, or should we just hardcode the value?

File: sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts [hunk 2]
```diff
@@ -2,56 +2,54 @@ import { css } from "@microsoft/fast-element";
...
+
+    .list-item:hover .icon {
         fill: var(--accent-fill-hover);
     }
 
     .scroll-indicator {
-        height: var(--design-unit-large);
-        width: calc(var(--design-unit) * 1px);
-        background-color: ${neutralForegroundRestBehavior.var};
+        height: calc(var(--design-unit) * 9px);
```
Comment (chrisdholt) [inline sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts:38]: Similar question to above - does this change with the design unit, or should it just be hardcoded?

File: sites/fast-website/src/app/css/style.css [hunk 3]
```diff
@@ -31,34 +31,23 @@ body {
...
+  width: 100%;
+  height: min-content;
   position: fixed;
-  top: 40%;
-  right: 0%;
+  top: 50%;
+  transform: translateY(-50%);
+  display: flex;
+  justify-content: space-between;
+  z-index: -1;
```
Comment (chrisdholt) [inline sites/fast-website/src/app/css/style.css:42]: Do we need z-index here? Why do we need to bury this container?

File: sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts (hunk 1 above)
Comment (SamanthaAO) [reply sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts:11]: @chrisdholt My thinking was that if I used design unit as part of the height that then the scroll indicators would adjust with the rest of the design system as we adjusted the page's density. If this is not the case I will hard code them.

Comment (chrisdholt) [reply sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts:11]: @SamanthaAO I'm primarily wondering if we've considered or seen how vast that difference can be. Could this end up obstructing the UI, etc. Technically, I'm not sure changing the density multiplier would directly affect this. Changing the design unit certainly would. A small change to the design unit could introduce significant changes here. I think we could certainly look at making these adaptive, I'm just not sure that we have enough insight into how we should go about that now.

File: sites/fast-website/src/app/css/style.css (hunk 3 above)
Comment (radium-v) [reply sites/fast-website/src/app/css/style.css:42]: Without the `z-index`, the container sits on top of the content and prevents interacting with any content in the center of the page.

File: sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts (hunk 1 above)
Comment (SamanthaAO) [reply sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts:11]: That makes sense. I think that the safest decision now would be to hard code these values. I will update it.

**Pull request merged**